project-2/Inconsisent_Data.ipynb

```diff
@@ -0,0 +1,5135 @@
+{
```

Comment:
<p>For the last 20 or so cells, you've been working without letting us know what's going on. No comments, no markdown descriptions, just computer logic back to back to back. Let's fix that</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Data-Epic/Humphery_Ufuoma/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
i have added more comments now and markdowns to explain my line of actions there